docs/examples/echo_server.md

```diff
@@ -143,8 +144,7 @@ In this walkthrough, you'll
                 alb.ingress.kubernetes.io/tags: Environment=dev,Team=test
         spec:
             rules:
-            - host: echoserver.example.com
-                http:
+            - http:
                 paths:
```

Comment:
The indentation is off here. Could you fix it?

Reply:
It was actually off before. I fixed it here. Reference - https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#ingressrule-v1-networking-k8s-io